app/liquid_tags/comment_tag.rb

```diff
@@ -8,13 +8,12 @@ def initialize(_tag_name, id_code, _tokens)
 
   def render(_context)
     raise_error unless @comment
-    commentable = if @comment.commentable_type == "Article"
-                    Article.find_by_id(@comment.commentable_id)
-                  else
-                    PodcastEpisode.find_by_id(@comment.commentable_id)
-                  end
+    if @comment.commentable_type == "Article"
+      Article.find_by_id(@comment.commentable_id)
+    else
+      PodcastEpisode.find_by_id(@comment.commentable_id)
+    end
```

Comment:
If I'm not mistaken, this entire block is useless. I believe a past version of this code probably used `commentable` later in the method.

Reply:
oops. thanks your review!

Reply:
I believe this was left over from a previous implementation where the "path" was defined right here based on the commentable. The "path" method changed to a method on the comment, and this code was left behind. So yeah, it can be removed.